ydb/core/kqp/executer_actor/kqp_scheme_executer.cpp

```diff
@@ -117,7 +118,8 @@ class TKqpSchemeExecuter : public TActorBootstrapped<TKqpSchemeExecuter> {
                 auto modifyScheme = schemeOp.GetDropTable();
                 if (Temporary) {
                     auto* dropTable = modifyScheme.MutableDrop();
-                    dropTable->SetName(dropTable->GetName() + SessionId);
+                    dropTable->SetTemporary(true);
+                    dropTable->SetSessionActorId(Target.ToString());
```

Comment:
What we need here is to make it more abstract, let's say OwnerId. I assume not only sessions will create new temporary tables in the future.